x-pack/plugins/apm/public/components/app/service_overview/service_overview.test.tsx

```diff
@@ -72,8 +72,7 @@ describe('ServiceOverview', () => {
         error_groups: [],
         total_error_groups: 0,
       },
-      // eslint-disable-next-line @typescript-eslint/naming-convention
-      'GET /api/apm/services/{serviceName}/overview_transaction_groups': {
+      'GET /api/apm/services/{serviceName}/transactions/groups/overview': {
```

Comment:
@dgieselaar have you merged upstream? I have made some changes to the API structure, for instance this API is no longer inside `/service_overview` it's now inside `/transactions`. https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/server/routes/transactions/transactions_routes.ts#L62

Reply:
Ah, you did never mind!!! Sorry!!